app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -123,14 +124,15 @@ private boolean delete(Media media, String reason) {
 
     private boolean showDeletionNotification(Context context, Media media, boolean result) {
         String message;
-        String title = "Nominating for Deletion";
+        String title = context.getString(R.string.delete_helper_show_deletion_notification_title);
 
         if (result) {
-            title += ": Success";
-            message = "Successfully nominated " + media.getDisplayTitle() + " deletion.";
+            title += context.getString(R.string.delete_helper_show_deletion_notification_title_if);
+            message = context.getString(R.string.delete_helper_show_deletion_notification_message_if_1)+ " "
+                    + media.getDisplayTitle() + context.getString(R.string.delete_helper_show_deletion_notification_message_if_2);
         } else {
-            title += ": Failed";
-            message = "Could not request deletion.";
+            title += context.getString(R.string.delete_helper_show_deletion_notification_title_else);
```

Comment:
Ideally this string should be just "Failed" and operation shuld be ": "+[string]